helix-core/src/test/java/org/apache/helix/controller/rebalancer/waged/TestAssignmentMetadataStore.java

```diff
@@ -20,6 +20,7 @@
  */
 
 import java.util.Map;
+
```

Comment:
Remove

Reply:
This is the line between java lib imports and non-java lib imports.